### ISSUE TITLE
in my browser: NV Access Website Fails to Update Recent Posts and NVDA Downloads page

### ISSUE THREAD

**Issue by HBM301** [nvaccess/nvda#18251]
Open the 
[NV Access page](https://nvaccess.org/)  and move to the recent posts 
Sometimes, the NV Access website does not display the most recent posts, such as 'Blog in Process' articles, or the beta and rc of NVDA, Although I pressed F5 to reload the page
The download section for NVDA has not yet displayed the latest release, the version currently available on the download page remains NVDA 2024.4.2, and Recent Posts is still NVDA 2025.1beta10
Currently I'm using 2 browsers, Chrome and Brave, I have tried with these 2 browsers

**Comment by seanbudd** [comment]
Our content is being cached. This is intended behaviour.
Perform a hard refresh or wait some time for the cache to flush.
